lib/screens/authenticate/forgotpass.dart

```diff
@@ -21,90 +21,90 @@ class _ForgotPassState extends State<ForgotPass> {
-              child: Form(
-                key: _formKey,
-                child: SingleChildScrollView(
-                  child: Column(
-                    children: <Widget>[
-                      SizedBox(height: 20.0),
-                      TextFormField(
-                        decoration:
-                            textInputDecoration.copyWith(hintText: 'Email'),
-                        validator: (val) =>
-                            val.isEmpty ? 'Enter a valid Email' : null,
-                        onChanged: (val) {
-                          setState(() => email = val);
-                        },
-                      ),
-                      SizedBox(height: 20.0),
-                      RaisedButton(
-                        color: Colors.pink[400],
-                        child: Text(
-                          'Send Password Reset Link',
-                          style: TextStyle(color: Colors.white),
-                        ),
-                        onPressed: () async {
-                          if (_formKey.currentState.validate()) {
-                            setState(() => loading = true);
-                            try {
-                              setState(() {
-                                email = email.trim();
-                              });
-                              await _auth.resetPassword(email);
-                              setState(() => message = 'Email sent');
-                              setState(() => loading = false);
-                            } catch (e) {
-                              setState(() {
-                                switch (e.code) {
-                                  case 'ERROR_USER_NOT_FOUND':
-                                    message =
-                                        "User with this email doesn't exist.";
-                                    break;
-                                  case 'ERROR_USER_DISABLED':
-                                    message =
-                                        'User with this email has been disabled.';
-                                    break;
-                                  case 'ERROR_TOO_MANY_REQUESTS':
-                                    message =
-                                        'Too many requests. Try again later.';
-                                    break;
-                                  default:
-                                    message = 'An undefined Error happened.';
-                                }
-                              });
-                              setState(() => loading = false);
-                            }
-                          } else {
-                            setState(() => message = 'Incorrect Email');
-                          }
-                        },
-                      ),
-                      SizedBox(height: 12.0),
-                      RaisedButton(
-                        color: Colors.blue,
```

Comment:
For this also and all the other Colors. Also, give relevant names to these colors in main.app. This will help when we convert to dark theme